test.sh

```diff
@@ -1,37 +0,0 @@
-#!/bin/sh
-
-set -eu  # Strict shell, w/o print commands (set -x), w/o -o pipefail
-
-TARANTOOL_VERSION="${TARANTOOL_VERSION:-1.10}"
-
-# Setup tarantool repository.
-#
-# --repo-only: Setup the repository, but don't install tarantool.
-#              The tarantool executable is installed later,
-#              together with the development package.
-#
-# --type live: Setup so called 'live' repository with the latest
-#              tarantool versions of given ${TARANTOOL_VERSION}
-#              branch. We want to verify the module against the
-#              latest versions to spot problems earlier.
-curl -fsSL https://tarantool.io/installer.sh | \
-    sudo "VER=${TARANTOOL_VERSION}" bash -s - --repo-only --type live
-
-# Install tarantool executable, headers and other build/test
-# dependencies of the module.
-sudo apt-get install -y tarantool tarantool-dev libevent-dev libsasl2-dev --force-yes
-pip install --user python-daemon PyYAML six==1.9.0 msgpack-python gevent==1.1.2
-
-TARANTOOL_DIR=/usr/include cmake . -DCMAKE_BUILD_TYPE=Release
-
-# third_party/libmemcached/bootstrap.sh runs /usr/local/bin/shellcheck on
-# itself when the executable is found. Disabled all warnings it finds to allow
-# Travis-CI to pass.
-SHELLCHECK_OPTS="-e SC1091 -e SC2027 -e SC2034 -e SC2035 -e SC2086 -e SC2119"
-SHELLCHECK_OPTS="${SHELLCHECK_OPTS} -e SC2120 -e SC2124 -e SC2128 -e SC2129"
-SHELLCHECK_OPTS="${SHELLCHECK_OPTS} -e SC2145 -e SC2154 -e SC2155 -e SC2166"
-SHELLCHECK_OPTS="${SHELLCHECK_OPTS} -e SC2181 -e SC2193 -e SC2206"
-export SHELLCHECK_OPTS
```

Comment:
Is the dfd58d7fd523eae9591cd64617f76755568e6299 hack is not needed anymore? Maybe it is because I call `make internalso libmemcached` here, which (AFAIR) builds more targets than just `make`. Don't remember precisely, to be honest.

Just curious, not a real objection.

Reply:
Shellcheck is optional dependence in target `libmemcached`.
You don't need to suppress Shellcheck warnings if you don't have installed Shellcheck.
https://github.com/tarantool/libmemcached/blob/master/bootstrap.sh#L1855-L1857

Reply:
And also `libmemcached` target is broken, see https://github.com/tarantool/memcached/issues/80

Reply:
'Optional' is not quite right word here: if I have shellcheck installed in the system (for whatever reason) I cannot opt out from running it.

You dig here more than me: for what we may need the `libmemcached` target? Why we don't need it here?

Reply:
(I'll reopen to don't miss it, if you don't object.)

Reply:
> You dig here more than me: for what we may need the libmemcached target?

libmemcached used for building conformance test `memcapable` that used in test suite `test/capable`.

>  Why we don't need it here?

In commit message:
```
    There are also tests that run with target 'make test-memcached-capable' and memtier
    benchmark, but both blocked by [1] and [2].
    
    1. https://github.com/tarantool/memcached/issues/80
    2. https://github.com/tarantool/memcached/issues/81
```

Reply:
Thanks for the clarification!